storage.bs

```diff
@@ -251,6 +262,12 @@ anticipated that some APIs will be applicable to both <a>storage types</a> going
  <li><p>If <var>A</var>'s <a for="storage key">origin</a> is not <a>same origin</a> with
  <var>B</var>'s <a for="storage key">origin</a>, then return false.
 
+ <li><p>If <var>A</var>'s <a for="storage key">top-level site</a> is not <a>same site</a>
+ with <var>B</var>'s <a for="storage key">top-level site</a>, then return false.
+
+ <li><p>If <var>A</var>'s <a for="storage key">cross-site ancestry</a> is not equal to
+ <var>B</var>'s <a for="storage key">cross-site ancestry</a>, then return false.
```

Comment:
```suggestion
 <li><p>If <var>A</var>'s <a for="storage key">cross-site ancestry</a> is <var>B</var>'s
 <a for="storage key">cross-site ancestry</a>, then return false.
```

Reply:
I changed this to be "if A's cross-site ancestry is not B's cross-site ancestry" - this suggestion says if "A's cross-site ancestry is B's cross-site ancestry" return false. But let me know if I misinterpreted your feedback!